src/platform/src/main/java/org/geoserver/platform/resource/Paths.java

```diff
@@ -102,7 +102,41 @@ public static String path(String... path) {
         }
         return toPath(names);
     }
-
+    
+    // runtime flag which, if true, throws an error for the WARN characters
+    static final boolean STRICT_PATH = 
+            Boolean.valueOf(System.getProperty("STRICT_PATH", "true"));
```

Comment:
It is not really back-portable with a "false" setting, which I guess was the main point of the discussion?